modules/rtbhouseBidAdapter.js

```diff
@@ -96,30 +63,257 @@ export const spec = {
+      }
+    })
+  }
+  if (params.clickUrl) {
+    assets.push({
+      id: OPENRTB.NATIVE.ASSET_ID.CLICK_URL,
+      required: params.clickUrl.required ? 1 : 0,
+      data: {
+        type: OPENRTB.NATIVE.DATA_ASSET_TYPE.CLICK_URL
+      }
+    })
+  }
+  if (params.cta) {
+    assets.push({
+      id: OPENRTB.NATIVE.ASSET_ID.CTA,
+      required: params.cta.required ? 1 : 0,
+      data: {
+        type: OPENRTB.NATIVE.DATA_ASSET_TYPE.CTA_TEXT,
+        len: params.cta.len || undefined
+      }
+    })
+  }
+  return assets;
+}
+
+/**
+ * Produces an OpenRTB Native Ads Image object
+ */
+function mapNativeImage(image, type) {
+  const img = {type: type};
+  if (image.aspect_ratios) {
+    const ratio = image.aspect_ratios[0];
+    const minWidth = ratio.min_width || 100;
+    img.wmin = minWidth;
+    img.hmin = (minWidth / ratio.ratio_width * ratio.ratio_height);
+  }
+  if (image.sizes) {
+    const size = Array.isArray(image.sizes[0]) ? image.sizes[0] : image.sizes;
+    img.w = size[0];
+    img.h = size[1];
+  }
+  return img
+}
+
+/**
+ * Produces Prebid bidObject from OpenRTB Bid object
+ */
+function interpretBannerBid(serverBid) {
+  return {
+    requestId: serverBid.impid,
+    mediaType: BANNER,
+    cpm: serverBid.price,
+    creativeId: serverBid.adid,
+    ad: serverBid.adm,
+    width: serverBid.w,
+    height: serverBid.h,
+    ttl: 55,
+    netRevenue: true,
+    currency: 'USD'
+  }
+}
+
+/**
+ * Produces Prebid bidObject from OpenRTB Bid object
+ */
+function interpretNativeBid(serverBid) {
+  return {
+    requestId: serverBid.impid,
+    mediaType: NATIVE,
+    cpm: serverBid.price,
+    creativeId: serverBid.adid,
+    width: 1,
+    height: 1,
+    ttl: 55,
+    netRevenue: true,
+    currency: 'USD',
+    native: interpretNativeAd(serverBid.adm),
+  }
+}
+
+/**
+ * Produces Prebid bidObject.native from OpenRTB Native Ad Markup
+ */
+function interpretNativeAd(adm) {
+  const native = JSON.parse(adm).native;
+  const result = {
+    clickUrl: encodeURIComponent(native.link.url),
+    impressionTrackers: native.imptrackers
+  };
+  native.assets.forEach(asset => {
+    switch (asset.id) {
+      case OPENRTB.NATIVE.ASSET_ID.TITLE:
+        result.title = asset.title.text;
+        break;
+      case OPENRTB.NATIVE.ASSET_ID.IMAGE:
+        result.image = {
+          url: encodeURIComponent(asset.img.url),
+          width: asset.img.w,
+          height: asset.img.h
+        };
+        break;
+      case OPENRTB.NATIVE.ASSET_ID.ICON:
+        result.icon = {
+          url: encodeURIComponent(asset.img.url),
+          width: asset.img.w,
+          height: asset.img.h
+        };
+        break;
+      case OPENRTB.NATIVE.ASSET_ID.BODY:
+        result.body = asset.data.value;
+        break;
+      case OPENRTB.NATIVE.ASSET_ID.SPONSORED:
+        result.sponsoredBy = asset.data.value;
+        break;
+      case OPENRTB.NATIVE.ASSET_ID.CLICK_URL:
+        result.clickUrl = encodeURIComponent(asset.data.value);
```

Comment:
Seems to be lacking potential support for link.clicktrackers array; Probably good to add now
Similar for link.fallback

Reply:
Right, this is redundant. ClickUrl is handled on the beginning of function.
I don't see support for clicktrackers and fallback in Prebid, am I missing something?